style.css

```diff
@@ -96,22 +99,270 @@ header {
+.fa-linkedin,
+.fa-github,
+.fa-medium,
+.fa-twitter,
+.fa-angellist {
   color: #505f79;
 }
 
-.fa-github {
-  color: #505f79;
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 16px;
+  border: solid #e5e5e5 1px;
+  background-color: #fff;
+  justify-content: center;
+  padding: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 40px;
 }
 
-.fa-medium {
-  color: #505f79;
+.snapshot {
+  width: 100%;
 }
 
-.fa-twitter {
-  color: #505f79;
+h2 {
+  width: 295px;
+  height: 36px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
 }
 
-.fa-angellist {
+.detail {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  padding: 0;
+}
+
+.main {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-right: 10px;
+  padding-right: 12px;
+}
+
+.main2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin-right: 10px;
+  padding-right: 12px;
+  float: left;
+  list-style-type: disc;
+  color: #7a869a;
+}
+
+.paragraph {
+  margin-top: 15px;
+  padding-right: 16px;
+}
+
+.shor-descr {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+}
+
+.language {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  flex-direction: row;
+  text-align: center;
+  margin-bottom: 36px;
+  margin-top: 12px;
+  padding-left: 0;
+  gap: 12px;
+}
+
+.language li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  padding-left: 10px;
+  padding-right: 10px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.project {
+  margin-top: 114px;
+  gap: 50px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.button {
+  width: 124px;
+  height: 48px;
+  display: flex;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-bottom: 28px;
+}
+
+.see-btndark {
+  padding: 12px;
+  border-radius: 8px;
+  color: #396df2;
+  display: inline-block;
+  font-weight: 500;
+  font-size: 17px;
+  text-decoration: none;
+}
+
+.about_section {
+  background-color: #fff;
+  border-top-right-radius: 50px;
+  padding-top: 8%;
+  margin-top: 12px;
+  width: 100%;
+}
+
+.me {
+  width: 327px;
+  height: 144px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  text-align: center;
+}
+
+button {
+  border: 1px solid #6070ff;
+  padding: 12px;
+  border-radius: 8px;
+  color: #396df2;
+  margin-left: 16px;
+  margin-bottom: 30px;
+  background-color: #fff;
+}
```

Comment:
- [ ] Please I highly recommend adding `hover` effects to the `button` class per the [Figma Design Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=HhgzrHa7uTvTkdte-0) requirements.

This is a screenshot of the `button` requirements. 

![Screenshot (181)](https://user-images.githubusercontent.com/78104544/229258631-8171d02c-f4ca-4e3e-a7a3-aea74ae25c06.png)